tutorials/efas/introduction/dcs5_copper_constant_properties_electrothermal.i

```diff
@@ -0,0 +1,1444 @@
+    extra_vector_tags = 'ref'
+    correct_edge_dropping = true
+    # use_displaced_mesh = true
+  []
+  [thermal_contact_interface_inside_die_powder]
+    type = ModularGapConductanceConstraint
+    variable = temperature_inside_powder_lm
+    secondary_variable = temperature
+    primary_boundary = die_wall_left
+    primary_subdomain = inside_die_primary_subdomain
+    secondary_boundary = powder_right
+    secondary_subdomain = inside_powder_secondary_subdomain
+    gap_geometry_type = CYLINDER
+    gap_flux_models = 'thermal_conduction_wall_powder closed_thermal_interface_inside_die_powder'
+    extra_vector_tags = 'ref'
+    correct_edge_dropping = true
+    # use_displaced_mesh = true
+  []
+  [electrical_contact_interface_inside_die_powder]
+    type = ModularGapConductanceConstraint
+    variable = potential_inside_powder_lm
+    secondary_variable = potential
+    primary_boundary = die_wall_left
+    primary_subdomain = inside_die_primary_subdomain
+    secondary_boundary = powder_right
+    secondary_subdomain = inside_powder_secondary_subdomain
+    gap_geometry_type = CYLINDER
+    gap_flux_models = 'electrical_conduction_wall_powder closed_electric_interface_inside_die_powder'
+    extra_vector_tags = 'ref'
+    correct_edge_dropping = true
+    # use_displaced_mesh = true
+  []
+  [thermal_contact_interface_inside_die_top_punch]
+    type = ModularGapConductanceConstraint
+    variable = temperature_inside_top_punch_lm
+    secondary_variable = temperature
+    primary_boundary = die_wall_left
+    primary_subdomain = inside_die_primary_subdomain
+    secondary_boundary = top_punch_right
+    secondary_subdomain = inside_top_punch_secondary_subdomain
+    gap_geometry_type = CYLINDER
+    gap_flux_models = 'thermal_conduction_wall_top_punch closed_thermal_interface_inside_die_top_punch'
+    extra_vector_tags = 'ref'
+    correct_edge_dropping = true
+    # use_displaced_mesh = true
+  []
+  [electrical_contact_interface_inside_die_top_punch]
+    type = ModularGapConductanceConstraint
+    variable = potential_inside_top_punch_lm
+    secondary_variable = potential
+    primary_boundary = die_wall_left
+    primary_subdomain = inside_die_primary_subdomain
+    secondary_boundary = top_punch_right
+    secondary_subdomain = inside_top_punch_secondary_subdomain
+    gap_geometry_type = CYLINDER
+    gap_flux_models = 'electrical_conduction_wall_top_punch closed_electric_interface_inside_die_top_punch'
+    extra_vector_tags = 'ref'
+    correct_edge_dropping = true
+    # use_displaced_mesh = true
+  []
+[]
+
+[Materials]
+  [graphite_electro_thermal_properties]
+    type = ADGenericConstantMaterial
+    prop_names = 'graphite_density graphite_thermal_conductivity graphite_heat_capacity graphite_electrical_conductivity graphite_hardness'
+    prop_values = ' 1.82e3               81                           1.5e3                   5.88e4                           1.0' #from G535 datasheet
```

Comment:
General comment that I am noting once for future documentation and references - this should be a citable reference, if possible. It is unclear how one would find this and the others mentioned in this region of the file.

Reply:
I agree that a citable reference would be better; unfortunately all we have is the manufacturer data sheet. I will update this file with the web address